doc/javascript-target.md

```diff
@@ -68,7 +68,7 @@ The steps to create your parsing code are the following:
  
 You are now ready to bundle your parsing code as follows:
  - following webpack specs, create a webpack.config file
- - in the `webpack.config` file, exclude node.js only modules using: `node: { module: "empty", net: "empty", fs: "empty" }`
+ - in the `webpack.config` file, exclude node.js only modules using: `resolve: { fallback: { fs: false } }`
```

Comment:
Not everybody is keen to move to latest webpack, so suggest we say l71 is for webpack vn whilst l72 is for webpack vn+1

Reply:
Let me check whether or not the configuration works in the previous version of Webpack. Based on other people's conversations I found on Google, it doesn't work there either. It probably worked 2 major versions ago. But let me confirm.

Reply:
You are right. Someone else confirmed that the configuration fails only in the latest version of webpack. I will update this PR to reflect that.